pull_request_template.md

```diff
@@ -10,6 +10,9 @@
 ### Notion Card Links
 (Please provide links to any relevant Notion card(s) relevant to this PR.)
 
+### What have you done to QA this feature?
```

Comment:
Maybe add a link to this PR or some document with a list of examples for QA in case people need a refresher? 
This keeps the size of the PR template down but provides breadcrumbs for self-analysis.

Reply:
I like the idea of linking back to this PR to keep things more concise.